sdk/core/System.ClientModel/README.md

```diff
@@ -0,0 +1,59 @@
+# System.ClientModel shared client library for .NET
```

Comment:
```suggestion
# System.ClientModel client library for .NET
```

nit: Conventions

Reply:
This makes it sound like it's a client library.  Can we call it a Core library, or foundational library?  Or just "System.ClientModel library for .NET"?